### DISCUSSION TITLE
2023-10-26-dahang

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#165] [General]
# 我的自行车入门实践 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Life买了一辆大行 P8 很是喜欢，想来整理一个入门者自行车周边的最佳实践，希望可以给想骑车的小伙伴一些入门参考，哈哈有想抄作业的可以抄一抄。

https://tw93.fun/2023-10-06/dahang.html

**Comment by QuakeWang** [comment]
打算等年后也入手一脸自行车，虽然平时上班通勤用不到（步行五分钟），但还是想等春天了，骑车逛逛杭州 哈哈

**Reply by tw93** [reply]
骑车可以让人更快乐，哈哈，想象一下，小太阳、有风、有树木的地方骑着车
